cmd/thv-operator/pkg/registryapi/manager.go

```diff
@@ -76,6 +79,20 @@ func (m *manager) ReconcileAPIService(
 		}
 	}
 
+	// Ensure pgpass secret for PostgreSQL authentication if dbConfig provided.
```

Comment:
Assuming that the secret contains the pgpass file would avoid this additional logic to create the secret

Reply:
if we configured a secret with the pgpass file, this would not be needed. but you explained before the rationale behind the design, so that's ok